envoy/config/subscription.h

```diff
@@ -191,10 +191,10 @@ class UntypedConfigUpdateCallbacks {
    * being updated. Accepted changes have their version_info reflected in subsequent
    * requests.
    */
-  virtual void onConfigUpdate(
-      const Protobuf::RepeatedPtrField<envoy::service::discovery::v3::Resource>& added_resources,
-      const Protobuf::RepeatedPtrField<std::string>& removed_resources,
-      const std::string& system_version_info) PURE;
+  virtual void
+  onConfigUpdate(absl::Span<const envoy::service::discovery::v3::Resource* const> added_resources,
+                 const Protobuf::RepeatedPtrField<std::string>& removed_resources,
+                 const std::string& system_version_info) PURE;
```

Comment:
It fields odd to mix `absl::Span` and `Protobuf::RepeatedPtrField` here. Do you plan to change the other as well?

Reply:
I'm not against this (as it makes the code more consistent). However, it requires modifying more interfaces.
I suggest doing this in a followup PR.

Reply:
That's fine by me.